ansible/roles/upgrade/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- include_tasks: looped_tasks.yml
+  with_items:
+    - 2.3.5
+    - 2.4.9
+    - 2.5.7
+    - 2.6.4
+    - 2.7.2
+
+  loop_control:
+    loop_var: version
+
+- name: install redis server
+  become: true
+  apt:
+    name: redis-server
+    state: present
+    update_cache: yes
+
+- name: update board_report dataset
+  become: true
+  become_user: postgres
+  command: psql ckan_default -c "UPDATE resource SET format = 'CSV' WHERE package_id = '2c5c458b-8a2f-4319-bd6a-58fd00b70e74';"
+
+- name: "rename *_preview -> *_view in production.ini"
+  become: true
+  ini_file:
+    path: /etc/ckan/default/production.ini
+    section: "app:main"
+    option: ckan.plugins
+    value: stats text_view recline_view nhsengland_skin datastore datapusher adfs
+    backup: yes
+
+- name: add default views setting to production.ini
+  become: true
+  ini_file:
+    path: /etc/ckan/default/production.ini
+    section: "app:main"
+    option: ckan.views.default_views
+    value: recline_view
+    backup: yes
+
+- name: add default views setting to production.ini
+  become: true
+  ini_file:
+    path: /etc/ckan/default/production.ini
+    section: "app:main"
+    option: ckan.site_url
+    value: "http://{{ inventory_hostname }}"
```

Comment:
Should this be https? (looks like it sets the site_url, but I'm not 100% sure)

Reply:
tl;dr: No, we don't have TLS configured for test domains

This is honestly kinda hacky.  It sets the value of `ckan.site_url` to the hostname of the current host (of which there is only one for us).  In production this would never be necessary because we want to be working with the main domain.  However during testing I need to be working with the public hostname assigned by AWS and it's quite tedious to go and edit this one config setting on every test run just to make one of the CKAN services work so I included it here to save me 30s on each deploy!  Next up, this can't be HTTPS since we don't have TLS configured for the AWS assigned public hostnames (I also have to manually disable TLS in the nginx config for each test instance or it auto redirects).

Reply:
Great 👍